### ISSUE TITLE
data.signaling matrix error when using mouse database (Error in identifyOverExpressedGenes)

### ISSUE THREAD

**Issue by AHerik** [sqjin/CellChat#632]
Hi, I'm having an error similar to a previous user (https://github.com/sqjin/CellChat/issues/264). If I run CellChatDB.mouse, and I subset my cellchat object (subsetData), I get this error: 

```
Error in identifyOverExpressedGenes(cellchat_DMSO) : 
  Please check `object@data.signaling` and ensure that you have run `subsetData` and that the data matrix `object@data.signaling` looks OK 
```

When I look at the data.signalling matrix of my cellchat object, I see 2 rows:

```
> cellchat_DMSO@data.signaling
2 x 10547 sparse Matrix of class "dgCMatrix"
  [[ suppressing 66 column names ‘AAACCCAAGAATTCAG-1’, ‘AAACCCACAACCCTCT-1’, ‘AAACCCACACCAGCCA-1’ ... ]]
                                                                                                                                                
C3 . . . . . . . . . .        . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 2.183826 . . . . . .
F2 . . . . . . . . . 2.642966 . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .        . . . . . .
                    
C3 . .        ......
F2 . 0.943755 ......

 .....suppressing 10481 columns in show(); maybe adjust 'options(max.print= *, width = *)'
 ..............................
```

I do not get this same error when I use CellChatDB.human. How can I fix my data.signalling matrix? 

Thank you for your help!

**Comment by sqjin** [comment]
If you work on the human data, you should use CellChatDB.human instead of
mouse database

**Comment by AHerik** [comment]
Hello Dr. @sqjin,

We are using mouse data, which is why I'm confused as to why it works with the CellChatDB.human instead.

Thank you for your swift reply, your tool is fantastic!

**Comment by heleof23** [comment]
@AHerik Hello! Were you able to resolve the issue? I can't seem to find any admin solution here, seems like i have to change the meta data.

**Comment by AHerik** [comment]
@heleof23 hello! Yes, I found the solution. In our case, it was twofold. Number one, our mouse genes were named differently from the CellChatDB gene names, and I changed the meta data to have the same format of gene names. Number two, I was actually using reindeer data in another one of our projects, and it worked when I used the human cell chat db because our gene names were orthologous with the human equivalents, and were named that way. 

In short, the gene names have to match between your dataset and the cell chat dataset. Let me know if you need any help!
